overlord/snapstate/flags.go

```diff
@@ -98,6 +98,9 @@ type Flags struct {
 	// "per-snap" in case each snap is treated in a different
 	// transaction.
 	Transaction client.TransactionType `json:"transaction,omitempty"`
+
+	// QuotaGroupName
+	QuotaGroupName string `json:"quota-group-name,omitempty"`
```

Comment:
same

Reply:
Fixed here as well, and updated the doc string which was empty